packages/client/components/ReflectionGroup/RemoteReflection.tsx

```diff
@@ -273,6 +275,16 @@ const RemoteReflection = (props: Props) => {
             editorState={editorState}
             readOnly
             disableAnonymity={disableAnonymity}
+            teamId={teamId}
+            editorRef={editorRef}
+            placeholder=''
+            setEditorState={() => {
```

Comment:
+1 could this and `handleKeyDownFallback` and `placeholder` be optional props in `ReflectionEditorWrapper`?